satellite-clone-vars.sample.yml

```diff
@@ -28,9 +28,8 @@
 # uncomment the following line to override this option (defaults to true)
 #run_pre_install_check: true
 
-# Use register_to_portal to automatically register to Red Hat Portal. By default,
-#   - the host will not be registered to RH portal.
-#   - it is the responsibility of the user to make the required repos available.
+# Use register_to_portal to automatically register to Red Hat Portal. By default,the host will not
```

Comment:
looks like a space is missing `default,the host`

Reply:
host is also confusing here (makes me think of a Satellite host), could we change that to "system"?